CMakeLists.txt

```diff
@@ -212,4 +212,3 @@ install(FILES
 
 #install the cmake finder
 install(EXPORT s2nTargets DESTINATION "${INSTALL_CMAKE_DIR}/s2n" COMPONENT dev)
-
```

Comment:
Why the white space change?

Reply:
Must have been my editor, let me change this quick